data_science/lib/cohort_data.rb

```
@@ -0,0 +1,17 @@
+require 'json'
+
+class CohortData
+  attr_accessor :cohort_visits, :file_name
+
+  def initialize(file_name)
+    @file_name = file_name
+  end
+
+  def load_json_data_into_array
+    puts "File #{file_name} not found" unless File.exist?(file_name)
```

Line 11: At this point you want to raise an error `raise "FIle #{filename} not found" unless File.exist?(file_name)"`